src/tests/ftest/server/multiengine_persocket.py

```diff
@@ -135,21 +136,19 @@ def daos_server_storage_prepare_ns(self, step, engines_per_socket=1):
         cmd.sudo = False
         cmd.debug.value = False
         cmd.set_sub_command("scm")
-        cmd.sub_command_class.set_sub_command("reset")
+        cmd.sub_command_class.set_sub_command("prepare")
```

Comment:
It will be nice to implement the sub-commands under daos_utils.py, daos_utils_base.py. In this way, other tests cases can utilize the subcommands as needed. Another enhancement to be considered for future.

Reply:
Yes. This test to make sure the first multiple-engine per socket tests stabilize, and could be update to ../ftest/util/. in the future.

Reply:
+1 to Ravi's comment and we should also move to use `run_remote()` instead of `run_pcmd()` for ease of use.